File: 05-greatest-common-factor/05-greatest-common-factor.py
def findGCD(n1, n2):
    bigger_num = max(n1, n2)
    smaller_num = min(n1, n2)
    while bigger_num % smaller_num != 0:
        bigger_num, smaller_num = smaller_num, bigger_num % smaller_num
    return smaller_num

File: 05-greatest-common-factor/05-greatest-common-factor.js
function findGCD(n1, n2) {
  let biggerNum = Math.max(n1, n2);
  let smallerNum = Math.min(n1, n2);
  while (biggerNum % smallerNum != 0) {
    [biggerNum, smallerNum] = [smallerNum, biggerNum % smallerNum];
  }
  return smallerNum;
}
